### ISSUE TITLE
[Bug] Artist Names Appear As Null On Homepage After Updating To New Version

### ISSUE THREAD

**Issue by DarkCrypt** [gokadzev/Musify#183]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to latest version **[Latest](https://github.com/gokadzev/Musify/releases)**.
- [X] I will fill out all

### Current Behavior

When updating app from v4.7.0 -> v4.8.0, all artist names appear as null on homepage.
See screenshot.

![Screenshot-Musify Homepage Artist Names Null](https://user-images.githubusercontent.com/121076677/223911045-902632f1-3186-46c4-8dc7-a013716eb872.jpg)

### Expected Behavior

Artist names should appear properly instead of null after updating to new version.

### Steps To Reproduce

Reproduce:
1. Download new version(v4.8.0) from official release page or from in app updater.
2. Update old version to new version from download folder.
3. Open Musify and artist names appear as null on homepage.

### Musify Version

V4.8.0

### Operating System Version

Android 13

### Additional information

Issue is non existent with fresh install of v4.8.0, only after update process from v4.7.0.

**Comment by gokadzev** [comment]
@DarkCrypt It should be different caches problem, since i made some changes/cleanup in structure. try to clean cache with app in-built feature and reopen the app or reinstall it

**Comment by DarkCrypt** [comment]
Ok, all I needed to do was hit clear cache in app. Then went back to homepage and it refreshed with artist names properly displayed. Sorry, I need to remember clearing cache can fix issues sometimes.
